importScripts/sampleRecords/locationHistory.destination.json

```diff
@@ -335,11 +315,11 @@
       "ca_objects.type_id":"Artwork",
       "locationHierarchy":"Artworks room;B3",
       "ca_storage_locations":"B3",
-      "ca_storage_locations.type_id":"container"
+      "ca_storage_locations.type_id":"shelf"
```

Comment:
back in sync from here. You can see it's been replaced the default of `container` with `shelf`